2020-02-28 19:35:56,640 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 160, callback <bound method Player.heartbeat of <player.Player object at 0x7fd68f219470>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 272, in heartbeat
    cmd = self.cons.take_input()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/console.py", line 518, in take_input
    self.upload_file(self.file_input)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/console.py", line 405, in upload_file
    f = open(self.current_directory+'/'+self.uploading_filename, 'r')
IsADirectoryError: [Errno 21] Is a directory: 'home/scott/'
2020-02-28 19:36:00,433 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 161, callback <bound method Player.heartbeat of <player.Player object at 0x7fd68f219470>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 272, in heartbeat
    cmd = self.cons.take_input()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/console.py", line 518, in take_input
    self.upload_file(self.file_input)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/console.py", line 405, in upload_file
    f = open(self.current_directory+'/'+self.uploading_filename, 'r')
IsADirectoryError: [Errno 21] Is a directory: 'home/scott/'

=====================================================================================================

2020-02-28 20:29:41,483 - : - WARNING - Gitbot failed to accept POST request
2020-02-28 20:33:54,809 - : - WARNING - Gitbot failed to accept POST request
2020-02-28 20:34:42,876 - : - WARNING - Gitbot failed to accept POST request
2020-02-28 20:37:41,127 - : - WARNING - Gitbot failed to accept POST request
2020-02-28 20:39:29,422 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 755, callback <bound method Player.heartbeat of <player.Player object at 0x7f4d1f05b4e0>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 282, in heartbeat
    self.detach()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 10:22:32,398 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:24:02,467 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:26:12,665 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:29:42,021 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:30:00,078 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:31:29,232 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:32:27,663 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 803, callback <bound method Player.heartbeat of <player.Player object at 0x7fb5cc085400>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 282, in heartbeat
    self.detach()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 10:54:23,965 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:55:09,043 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:56:47,218 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 10:58:30,319 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:00:00,400 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:00:41,538 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:01:05,551 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:03:14,799 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:03:24,823 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:04:48,566 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 630, callback <bound method Player.heartbeat of <player.Player object at 0x7f8d75556470>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 282, in heartbeat
    self.detach()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:05:20,692 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 662, callback <bound method Player.heartbeat of <player.Player object at 0x7f8d75556470>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 282, in heartbeat
    self.detach()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:05:28,564 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 670, callback <bound method Player.heartbeat of <player.Player object at 0x7f8d75450a90>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 272, in heartbeat
    cmd = self.cons.take_input()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/console.py", line 531, in take_input
    internal = self._handle_console_commands()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/console.py", line 391, in _handle_console_commands
    self.user.emit("&nD%s fades from view, as if by sorcery...you sense that &p%s is no longer of this world." % (self.user.id, self.user.id))
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/thing.py", line 396, in emit
    holder.perceive(message)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 370, in perceive
    if not self.location.is_dark() or force:
AttributeError: 'NoneType' object has no attribute 'is_dark'
2020-03-01 11:05:49,616 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 691, callback <bound method Player.heartbeat of <player.Player object at 0x7f8d75362160>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 272, in heartbeat
    cmd = self.cons.take_input()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/console.py", line 531, in take_input
    internal = self._handle_console_commands()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/console.py", line 391, in _handle_console_commands
    self.user.emit("&nD%s fades from view, as if by sorcery...you sense that &p%s is no longer of this world." % (self.user.id, self.user.id))
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/thing.py", line 396, in emit
    holder.perceive(message)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 370, in perceive
    if not self.location.is_dark() or force:
AttributeError: 'NoneType' object has no attribute 'is_dark'
2020-03-01 11:10:43,923 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 16, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:44,924 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 17, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:45,928 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 18, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:46,928 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 19, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:47,930 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 20, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:48,931 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 21, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:49,933 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 22, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:50,935 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 23, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:51,996 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 24, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:52,937 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 25, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:53,938 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 26, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:54,940 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 27, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:55,943 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 28, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:56,945 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 29, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:57,946 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 30, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:58,948 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 31, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:10:59,949 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 32, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:00,952 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 33, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:01,954 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 34, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:02,956 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 35, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:03,957 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 36, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:04,959 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 37, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:05,962 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 38, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:06,964 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 39, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:07,965 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 40, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:08,966 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 41, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:09,968 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 42, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:10,970 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 43, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:11,972 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 44, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:12,973 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 45, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:13,975 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 46, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:14,977 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 47, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:15,978 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 48, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:16,980 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 49, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:17,981 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 50, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:18,983 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 51, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:19,984 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 52, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:20,985 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 53, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:21,986 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 54, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:22,987 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 55, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:23,989 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 56, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:24,991 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 57, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:25,992 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 58, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:26,994 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 59, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:27,995 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 60, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:28,996 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 61, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:29,996 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 62, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:31,000 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 63, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:32,001 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 64, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:33,004 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 65, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:34,007 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 66, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:35,008 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 67, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:36,009 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 68, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:46,135 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 69, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:49,409 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:11:49,429 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 70, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:50,431 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 71, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:51,433 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 72, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:52,434 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 73, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:53,437 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 74, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:54,438 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 75, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:55,440 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 76, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:56,441 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 77, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:57,442 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 78, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:58,443 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 79, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:11:59,445 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 80, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:00,448 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 81, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:01,449 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 82, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:02,449 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 83, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:03,452 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 84, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:04,452 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 85, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:17,542 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 86, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:44,378 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 87, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:52,030 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 88, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:53,027 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 89, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:54,029 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 90, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:55,030 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 91, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:56,032 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 92, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:57,033 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 93, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:58,034 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 94, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:12:59,036 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 95, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:00,037 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 96, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:01,039 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 97, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:02,041 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 98, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:03,044 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 99, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:04,045 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 100, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:05,046 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 101, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:06,050 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 102, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:07,051 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 103, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:08,053 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 104, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:09,054 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 105, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:10,056 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 106, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:11,059 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 107, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:12,061 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 108, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:13,062 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 109, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:14,063 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 110, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:15,065 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 111, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:16,066 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 112, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:17,067 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 113, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:18,068 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 114, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:19,070 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 115, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:20,073 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 116, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:21,073 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 117, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:22,075 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 118, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:23,077 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 119, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:24,078 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 120, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:25,080 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 121, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:26,082 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 122, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:27,084 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 123, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:28,085 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 124, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:29,087 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 125, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:30,089 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 126, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:31,091 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 127, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:32,092 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 128, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:33,093 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 129, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:34,094 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 130, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:35,095 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 131, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:36,096 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 132, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:37,097 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 133, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:38,098 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 134, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:39,100 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 135, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:40,101 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 136, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:41,103 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 137, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:42,106 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 138, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:43,107 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 139, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:44,109 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 140, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:45,111 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 141, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:46,112 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 142, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:47,114 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 143, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:48,117 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 144, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:49,119 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 145, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:50,120 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 146, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:51,121 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 147, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:52,122 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 148, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:53,124 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 149, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:54,126 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 150, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:55,127 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 151, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:56,129 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 152, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:57,130 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 153, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:58,131 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 154, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:13:59,134 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 155, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:00,135 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 156, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:01,137 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 157, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:02,139 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 158, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:03,141 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 159, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:04,144 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 160, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:05,146 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 161, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:06,150 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 162, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:07,151 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 163, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:08,153 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 164, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:09,154 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 165, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:10,156 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 166, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:11,158 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 167, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:12,163 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 168, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:13,163 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 169, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:14,169 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 170, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:15,170 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 171, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:16,175 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 172, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:17,176 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 173, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:18,181 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 174, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:19,182 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 175, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:20,184 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 176, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:21,185 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 177, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:22,189 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 178, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:23,192 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 179, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:24,193 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 180, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:25,194 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 181, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:26,196 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 182, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:27,198 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 183, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:28,200 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 184, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:14:29,201 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 185, callback <bound method Player.heartbeat of <player.Player object at 0x7f0993859438>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 267, in heartbeat
    self.detach(nocons=True)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 11:17:46,227 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:18:25,100 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 11:20:25,186 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 12:18:46,411 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 12:24:02,688 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 12:34:18,854 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 774, callback <bound method Player.heartbeat of <player.Player object at 0x7fa81e964400>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 282, in heartbeat
    self.detach()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 12:34:19,718 - _gameserver - CRITICAL - Exiting main game loop!
2020-03-01 12:34:59,092 - : - WARNING - Gitbot failed to accept POST request
2020-03-01 12:36:12,256 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 103, callback <bound method Player.heartbeat of <player.Player object at 0x7f200bce3390>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 282, in heartbeat
    self.detach()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 12:37:20,445 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 171, callback <bound method Player.heartbeat of <player.Player object at 0x7f200bce3390>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 282, in heartbeat
    self.detach()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 12:37:21,411 - _gameserver - CRITICAL - Exiting main game loop!
2020-03-01 12:45:10,964 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 72, callback <bound method Player.heartbeat of <player.Player object at 0x7f9503f74630>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 402, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 282, in heartbeat
    self.detach()
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 257, in detach
    del dbg.verbosity[self.id]
NameError: name 'dbg' is not defined
2020-03-01 12:45:11,792 - _gameserver - CRITICAL - Exiting main game loop!
2020-03-06 20:30:27,185 - : - ERROR - Error! Location is a string!
2020-03-06 20:30:56,668 - : - ERROR - Error! Location is a string!
2020-03-07 09:53:36,009 - : - ERROR - scott668163-saveplayer142812 was already removed from Thing.ID_dict
2020-03-07 09:53:36,014 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 12, callback <bound method Player.heartbeat of <player.Player object at 0x7fd88083a748>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/gameserver.py", line 425, in catch_func_errs
    func(*params)
  File "/mnt/c/Users/Owen Luebke/Programming/GitHub/old-skool-text-game/player.py", line 274, in heartbeat
    cmd = self.cons.take_input()
AttributeError: 'NoneType' object has no attribute 'take_input'
2020-04-03 21:07:13,640 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 187, callback <bound method Player.heartbeat of <player.Player object at 0x7f112fa921d0>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/src/old-skool-text-game/gameserver.py", line 522, in catch_func_errs
    func(*params)
  File "/src/old-skool-text-game/player.py", line 274, in heartbeat
    cmd = self.cons.take_input()
  File "/src/old-skool-text-game/console.py", line 449, in take_input
    internal = self._handle_console_commands()
  File "/src/old-skool-text-game/console.py", line 251, in _handle_console_commands
    self.download_file(path, edit_flag=True)
  File "/src/old-skool-text-game/console.py", line 365, in download_file
    f = open(gametools.realdir(filename, player=self.user.name()), 'rb')
AttributeError: module 'gametools' has no attribute 'realdir'
2020-04-03 21:07:30,671 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 204, callback <bound method Player.heartbeat of <player.Player object at 0x7f112fa921d0>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/src/old-skool-text-game/gameserver.py", line 522, in catch_func_errs
    func(*params)
  File "/src/old-skool-text-game/player.py", line 274, in heartbeat
    cmd = self.cons.take_input()
  File "/src/old-skool-text-game/console.py", line 449, in take_input
    internal = self._handle_console_commands()
  File "/src/old-skool-text-game/console.py", line 251, in _handle_console_commands
    self.download_file(path, edit_flag=True)
  File "/src/old-skool-text-game/console.py", line 365, in download_file
    f = open(gametools.realdir(filename, player=self.user.name()), 'rb')
AttributeError: module 'gametools' has no attribute 'realdir'
2020-04-03 21:07:37,681 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 211, callback <bound method Player.heartbeat of <player.Player object at 0x7f112fa921d0>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/src/old-skool-text-game/gameserver.py", line 522, in catch_func_errs
    func(*params)
  File "/src/old-skool-text-game/player.py", line 274, in heartbeat
    cmd = self.cons.take_input()
  File "/src/old-skool-text-game/console.py", line 449, in take_input
    internal = self._handle_console_commands()
  File "/src/old-skool-text-game/console.py", line 251, in _handle_console_commands
    self.download_file(path, edit_flag=True)
  File "/src/old-skool-text-game/console.py", line 365, in download_file
    f = open(gametools.realdir(filename, player=self.user.name()), 'rb')
AttributeError: module 'gametools' has no attribute 'realdir'
2020-04-03 21:07:40,686 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 214, callback <bound method Player.heartbeat of <player.Player object at 0x7f112fa921d0>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/src/old-skool-text-game/gameserver.py", line 522, in catch_func_errs
    func(*params)
  File "/src/old-skool-text-game/player.py", line 274, in heartbeat
    cmd = self.cons.take_input()
  File "/src/old-skool-text-game/console.py", line 449, in take_input
    internal = self._handle_console_commands()
  File "/src/old-skool-text-game/console.py", line 251, in _handle_console_commands
    self.download_file(path, edit_flag=True)
  File "/src/old-skool-text-game/console.py", line 365, in download_file
    f = open(gametools.realdir(filename, player=self.user.name()), 'rb')
AttributeError: module 'gametools' has no attribute 'realdir'
2020-04-03 21:09:09,249 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 243, callback <bound method Player.heartbeat of <player.Player object at 0x7f112fa921d0>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/src/old-skool-text-game/gameserver.py", line 522, in catch_func_errs
    func(*params)
  File "/src/old-skool-text-game/player.py", line 274, in heartbeat
    cmd = self.cons.take_input()
  File "/src/old-skool-text-game/console.py", line 449, in take_input
    internal = self._handle_console_commands()
  File "/src/old-skool-text-game/console.py", line 251, in _handle_console_commands
    self.download_file(path, edit_flag=True)
  File "/src/old-skool-text-game/console.py", line 365, in download_file
    f = open(gametools.realDir(filename, player=self.user.name()), 'rb')
AttributeError: module 'gametools' has no attribute 'realdir'
2020-04-03 21:13:50,358 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:13:55,355 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:14:15,408 - _gametools - ERROR - Error importing room module home.fastar.entryway
2020-04-03 21:14:15,426 - _gametools - ERROR - Error importing room module home.cedric.blank_room
2020-04-03 21:14:20,401 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:14:31,426 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:14:40,439 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:14:40,441 - _gametools - ERROR - Error importing module domains.school.school.library
2020-04-03 21:14:50,457 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:14:54,509 - : - ERROR - Error! Location is a string!
2020-04-03 21:15:15,767 - : - ERROR - scott668163-saveplayer978950 was already removed from Thing.ID_dict
2020-04-03 21:15:15,769 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 15, callback <bound method Player.heartbeat of <player.Player object at 0x7f85bb890240>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/src/old-skool-text-game/gameserver.py", line 522, in catch_func_errs
    func(*params)
  File "/src/old-skool-text-game/player.py", line 274, in heartbeat
    cmd = self.cons.take_input()
AttributeError: 'NoneType' object has no attribute 'take_input'
2020-04-03 21:15:28,795 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:15:45,822 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:17:14,942 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:20:03,931 - _gametools - ERROR - Error importing room module domains.school.school.library
2020-04-03 21:20:08,961 - _gametools - ERROR - Error importing module domains.school.school.library_bookcase
2020-04-03 21:23:33,361 - _gameserver - ERROR - An error occured while attepting to complete event (timestamp 415, callback <bound method Player.heartbeat of <player.Player object at 0x7f85bb8a5b00>>, payload [])! Printing below:
Traceback (most recent call last):
  File "/src/old-skool-text-game/gameserver.py", line 522, in catch_func_errs
    func(*params)
  File "/src/old-skool-text-game/player.py", line 274, in heartbeat
    cmd = self.cons.take_input()
  File "/src/old-skool-text-game/console.py", line 449, in take_input
    internal = self._handle_console_commands()
  File "/src/old-skool-text-game/console.py", line 312, in _handle_console_commands
    if len(self.words) > 1 and self.words[1] == 'game' and self.game.is_admin(self.user.name()):
AttributeError: 'Game' object has no attribute 'is_admin'
